src/psyclone/tests/domain/gocean/kernel/gokern_test.py

```diff
@@ -193,15 +161,8 @@ def test_gok_access_info_scalar_and_property():
             "p_fld%grid%subdomain%internal%xstop: READ, "
             "p_fld%grid%tmask: READ, p_fld: READWRITE")
 
-    # Check that the derived type using tmask has the corresponding component
-    # indices specified. No indices for p_fld and grid:
+    # Kernel calls have the whole field provided, so no indices are given
+    # at this level.
     tmask = vam[Signature("p_fld%grid%tmask")]
-    comp_ind = tmask[0].component_indices
-    assert comp_ind[0] == []
-    assert comp_ind[1] == []
-
-    # And it should have PSyIR expressions for (i,j) as the last component:
-    assert isinstance(comp_ind[2][0], Reference)
-    assert isinstance(comp_ind[2][1], Reference)
-    assert comp_ind[2][0].symbol.name == "i"
-    assert comp_ind[2][1].symbol.name == "j"
+    comp_ind = tmask[0].component_indices()
+    assert not comp_ind
```

Comment:
I have to admit I had no idea that `tuple(tuple())` is empty :)
It might be a bit clearer to use `assert comp_ind is tuple(tuple())`?